lib/kube/proxy/forwarder.go

```diff
@@ -255,6 +255,14 @@ func NewForwarder(cfg ForwarderConfig) (*Forwarder, error) {
 
 	fwd.router.UseRawPath = true
 
+	fwd.router.GET("/version", fwd.withAuth(
+		func(ctx *authContext, w http.ResponseWriter, r *http.Request, _ httprouter.Params) (interface{}, error) {
```

Comment:
nit: (We're using Go 1.20 on all production branches thanks to @codingllama)
```suggestion
		func(ctx *authContext, w http.ResponseWriter, r *http.Request, _ httprouter.Params) (any, error) {
```

Reply:
Not on v10 yet, I keep having issues with golangci locally. :/